requirements.txt

```diff
@@ -7,7 +7,7 @@ lz4==3.0.2
 requests==2.27.1
 networkx==2.5
 python-dateutil==2.8.2
-protobuf~=3.14.0
+protobuf~=3.15.0
```

Comment:
@hackaugusto do you know why we had such an old dependency on protobuf? the current one is `3.19.4`

Reply:
Hey @jlprat , it was most likely a blunder. Thank you for updating this :)